docs/source/concepts/runner.rst

```diff
@@ -51,6 +51,8 @@ methods.
     runner.predict.run( MODEL_INPUT )
 
 
+.. _custom-runner:
```

Comment:
this is required because it appears that auto-section-labels plugin does not work well with sphinx-design plugin's `card`, where a ref link can not be pointing to a label generated by autosectionlabels.